src/app/views/users/_user.html.erb

```diff
@@ -11,8 +11,20 @@
           <%= link_to user.name, user_functions_path(user) %>
         </p>
         <p class="subtitle is-6">
-          @<%= user.username %>
+          <%= link_to user_github_account_url(user),  target: "_blank" do %>
+            @<%= user.username %>
+          <% end %>
         </p>
+        
+        <% unless current_user.nil? || user.id == current_user.id %> 
+          <% text = current_user.following?(user.id) ? "Unfollow" : "Follow" %>
```

Comment:
after converting this to a helper will be 
```suggestion
          <% text = follows?(user.id) ? "Unfollow" : "Follow" %>
```
and it will assume current user as the user we're asking about